client-sdk/ts-web/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    presets: [['@babel/preset-env', {targets: {node: 'current'}}], '@babel/preset-typescript'],
+};
```

Comment:
Please configure your editor to add newlines at the end of the file.

Reply:
Fixed those files, if it is standard for this repository it should probably be in a standardized [editorconfig](https://editorconfig.org/) config file along with other standards at the root of the repository